### PR TITLE
board-types.txt:reserve IDs for VIEWPRO

### PR DESCRIPTION
**Pull request by LCaiJie**

### CODE REVIEW

File: Tools/AP_Bootloader/board_types.txt [hunk 1]
```diff
@@ -381,6 +381,9 @@ AP_HW_ZeroOne_GNSS                   5602
 # IDs 7000-7099 reserved for CUAV
 AP_HW_CUAV-7-NANO                    7000
 
+# IDs 7100-7199 reserved for VIEWPRO
```
Comment (peterbarker) [inline Tools/AP_Bootloader/board_types.txt:384]:
```suggestion
# IDs 7100-7109 reserved for VIEWPRO
```

**Review by peterbarker — CHANGES_REQUESTED** [review]

File: Tools/AP_Bootloader/board_types.txt (hunk 1 above)
Comment (LCaiJie) [reply Tools/AP_Bootloader/board_types.txt:384]: Ok, I'll edit and push it.

**Review by peterbarker — APPROVED** [review]
LGTM

**Comment by rmackay9** [comment]
Could we fixup the commit messages?  I think they should start with "AP_HAL_ChibiOS:"

**Pull request merged**